manual/docs/css/custom.css

```diff
@@ -0,0 +1,788 @@
+@import url(http://fonts.googleapis.com/css?family=Cutive+Mono);
```

Comment:
for the man page we might want to include these fonts in case the user doesn't have internet connection, what do you think?